test/auction-reorg-test.js

```diff
@@ -395,7 +395,7 @@ describe('Auction', function() {
     });
   });
 
-  describe('Claim', function() {
+  describe('Claim Reorg', function() {
```

Comment:
I could be wrong here but I do not think this test re-org's the chain. In fact I think this test is a complete unaltered copy of the `Claim` test in the original file `auction-test.js` and could probably just be removed.

Reply:
ACK, but let's heed JJ's warning for now. This PR only changes the names of the describe blocks and leaves program execution as-is. We can open a new PR if this test is safe to remove.